### PR TITLE
Add HTML skeleton code

### PR DESCRIPTION
**Pull request by cpars**
This is the initial push of the html code.

### CODE REVIEW

**Review by ArtOfTheNiles — APPROVED** [review]
Looks good, thank you

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!--This meta tag provides a brief summary of the page content-->
+  <meta name="description" content="Movie MatchMaker">
+  <!--This meta tag specifies the name of the author of the webpage-->
+  <meta name="author" content="Bontrager, Casalme, Parsons">
+  <!--Keywords that are relevant to the page content-->
+  <meta name="keywords" content="Movies">
+  <title>Movie Matchmaker</title>
```
Comment (ccasalme) [inline index.html:12]: add css reset

**Review by ccasalme — CHANGES_REQUESTED** [review]
css reset

**Review by ArtOfTheNiles — APPROVED** [review]

**Comment by cpars** [comment]
Added the reset.css code

**Comment by ccasalme** [comment]
approved

**Review by ccasalme — APPROVED** [review]
looks good!

**Pull request merged**